tests/phpunit/tests/general/feedLinksExtra.php

```diff
@@ -0,0 +1,468 @@
+<?php
+/**
+ * Test feed_links_extra().
+ *
+ * @ticket 54713
+ *
+ * @group general
+ * @group template
+ *
+ * @covers ::feed_links_extra
+ */
+class Tests_General_FeedLinksExtra extends WP_UnitTestCase {
+	protected static $author;
```

Comment:
```suggestion
	/**
	 * Author ID.
	 *
	 * @var int
	 */
	protected static $author;
	
```

🔢 Applies throughout. 

As you are getting the IDs rather than the object, you may wish to consider the suffix `_id` on each of these to help the code be self documenting.